src/nvim/eval/typval.h

```diff
@@ -81,7 +81,8 @@ typedef struct {
   } data;
   CallbackType type;
 } Callback;
-#define CALLBACK_NONE ((Callback){ .type = kCallbackNone })
+#define CASTFREE_CALLBACK_NONE { .type = kCallbackNone }
```

Comment:
the correct solution but call this `CALLBACK_INIT` :]

Reply:
@bfredl Done, thank you for your input :)